potluck/src/App.js

```diff
@@ -1,24 +1,45 @@
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
```

Comment:
With the latest React release we do not have to explicitly import React in each file. Just an FYI for future projects, and it won't break anything :smiley: